test/Fitness-test.js

```diff
@@ -346,4 +346,160 @@ describe('Fitness', function() {
       flightsOfStairs: 16
```

Comment:
we would need reorganize Fitness-test.js file with beforeEach to include all data  at the top of the test file if we get time in the end:)